imagetest/test_suites/oslogin/ssh_test.go

```diff
@@ -180,6 +180,28 @@ func TestAgent(t *testing.T) {
 	}
 }
 
+func sshClient(t *testing.T, hostname string, config *ssh.ClientConfig) *ssh.Client {
+	t.Helper()
+	var client *ssh.Client
+	var err error
+
+	// Retry dialing target system for a minute.
+	for i := 0; i < 5; i++ {
+		client, err = ssh.Dial("tcp", hostname, sshConfig)
+		if err != nil {
+			time.Sleep(10 * time.Second)
+		} else {
+			defer client.Close()
```

Comment:
Shouldn't this be happening outside of `sshClient`? Otherwise the client is closed with the return on line 202 and unusable on 493

Reply:
yep, just changed it.